source/faq.rst

```diff
@@ -23,23 +23,24 @@ To adjust PodBrowser's access:
 
 #. In the :guilabel:`Manage your trusted applications` section:
 
-   * If PodBrowser is not listed as a trusted application:
+  * If PodBrowser is not listed as a trusted application:
   
-     #. At the bottom of the table enter:
+    #. At the bottom of the table enter:
 
         .. list-table::
+          :header-rows: 1
 
-           * - Application URL
+          * - Application URL
 
-             - Access modes
+            - Access modes
 
-           * - ``https://podbrowser.inrupt.com``
+          * - ``https://podbrowser.inrupt.com``
 
-             - ``Read``, ``Write``, ``Append``, and ``Control``
+            - ``Read``, ``Write``, ``Append``, and ``Control``
+    
+    #. Click :guilabel:`Add`.
 
-     #. Click :guilabel:`Add`.
-
-    * If PodBrowser is listed as a trusted application but with the
+  * If PodBrowser is listed as a trusted application but with the
```

Comment:
Just curious - was the shifting of the bullet points because of the css for bullet points indented too much?

Reply:
I can't remember exactly, but I believe yes. When I built the page, the indentation was off.